api/managementpb/dbaas/db_clusters.proto

```diff
@@ -102,20 +102,20 @@ message ListDBClustersResponse {
 
 message RestartDBClusterRequest {
   // Kubernetes cluster name.
-  string kubernetes_cluster_name = 1 [(validator.field) = {string_not_empty: true}];
+  string kubernetes_cluster_name = 1 [(validate.rules).string.min_len = 1];
   // PXC cluster name.
-  string name = 2 [(validator.field) = {string_not_empty: true}];
+  string name = 2 [(validate.rules).string.min_len = 1];
   // DB cluster type.
-  DBClusterType cluster_type = 3 [(validator.field) = {msg_exists: true}];
+  DBClusterType cluster_type = 3;
```

Comment:
@gen1us2k Not sure what was the reason of that vaildation. DBClusterType is enum, it can't to `not exist`

Reply:
It seems that I was wrong

Reply:
Added `[(validate.rules).enum.defined_only = true]` rule